src/main/java/ch/njol/skript/util/EnumUtils.java

```diff
@@ -43,55 +40,67 @@ public EnumUtils(final Class<E> c, final String languageNode) {
 		
 		this.c = c;
 		this.languageNode = languageNode;
-		
+
 		names = new String[c.getEnumConstants().length];
 		
-		Language.addListener(new LanguageChangeListener() {
-			@Override
-			public void onLanguageChange() {
-				validate(true);
-			}
-		});
+		Language.addListener(() -> validate(true));
 	}
 	
 	/**
 	 * Updates the names if the language has changed or the enum was modified (using reflection).
 	 */
-	final void validate(final boolean force) {
+	void validate(final boolean force) {
 		boolean update = force;
-		
-		final int newL = c.getEnumConstants().length;
-		if (newL > names.length) {
-			names = new String[newL];
+
+		int newLength = c.getEnumConstants().length;
```

Comment:
Ideally you'd introduce a variable for the Enum array here rather than querying it multiple times later in the method, but it's very minor and not essential

Reply:
If you introduced a variable for the constants you could discard this int variable since `.length` is a trivial opcode